src/consent-manager-builder/index.tsx

```diff
@@ -316,25 +316,22 @@ export default class ConsentManagerBuilder extends Component<Props, State> {
 
       const newDestinations = getNewDestinations(destinations, destinationPreferences)
 
-      // If preferences haven't changed, don't reload the page as it's a disruptive experience for end-users
-      if (prevState.havePreferencesChanged || newDestinations.length > 0) {
```

Comment:
Removing this `if` is not ideal as well. To test the difference: 
Work-in-progress branch: When opening the consent modal and clicking `save` the page will refresh even if we haven't changed anything. 
Master branch: When opening the consent modal and clicking `save` the page won't refresh unless there is a change.

Reply:
I agree with you, I did not realize that the page was refreshing when removing the if. This was fixed in the last commit.